lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1621,6 +1624,17 @@ public function getCreateTableSQL(Table $table, $createFlags = self::CREATE_INDE
         return array_merge($sql, $columnSql);
     }
 
+    public function getCommentOnTableSQL(string $tableName, ?string $comment) : string
```

Comment:
It is a good question. I actually tried to mimic what was done with the comments on columns to be consistent.

There is already a `public function getCommentOnColumnSQL()` method: https://github.com/doctrine/dbal/blob/4ea7c119d32d7fc1f7755cb5464a2f4f34e60fa4/lib/Doctrine/DBAL/Platforms/AbstractPlatform.php#L1639-L1656

If I was to start from zero, I would probably make the method protected, put it in a trait, and "use" this trait directly in the XXXSchemaManager classes. But since there is already a `getCommentOnColumnSQL` method, I thought it would be better to do something similar.

I'm really open to any change here, if you think there is a better way.

Reply:
It is useful to have it as a separate method because the statement can be different (typically, this method is overloaded for SQLServer). However, you are right that it is not necessary to have this "public". I'm moving this to "protected".